packages/examples/src/langium/wrapperLangium.ts

```diff
@@ -1,122 +1,43 @@
 import { buildWorkerDefinition } from 'monaco-editor-workers';
-buildWorkerDefinition('../../node_modules/monaco-editor-workers/dist/workers', import.meta.url, false);
+buildWorkerDefinition('../../../../node_modules/monaco-editor-workers/dist/workers', import.meta.url, false);
 
 import * as vscode from 'vscode';
 import { MonacoEditorLanguageClientWrapper } from 'monaco-editor-wrapper';
-import { BrowserMessageReader, BrowserMessageWriter, Message } from 'vscode-languageserver/browser.js';
+import { createLangiumGlobalConfig } from './langiumWrapperConfig.js';
 
-const client = new MonacoEditorLanguageClientWrapper(true);
+const wrapper = new MonacoEditorLanguageClientWrapper();
+const wrapper2 = new MonacoEditorLanguageClientWrapper();
```

Comment:
Two wrappers?

Reply:
I changed this to show a problem (#16) that we want to fix after v2 release. This PR is already super-large. I didn't want add more things.

Reply:
Are you ok with this explanation?

Reply:
Yep, no problem there